### PR TITLE
feature(ui-ux): implementation of asset list view for portfolio page

### PR DESCRIPTION
**Pull request by fullstackninja864**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [ ] No console errors on web
- [ ] Tested on Light mode and Dark mode*
- [ ] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

**Comment by kyleleow** [comment]
- To align token amount
- To extend background to viewport height
image

**Comment by kyleleow** [comment]
loan token icon has a border now
image

**Comment by fullstackninja864** [comment]
**Known issue:** Border for loan token are cutting from its edges, will fix this in next PR
@kyleleow

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/TokenAmountTextV2.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  denominationCurrency
+}: TokenAmountTextProps): JSX.Element {
+  const { isBalancesDisplayed, hiddenBalanceText } = useDisplayBalancesContext()
+
+  return (
+    <NumberFormat
+      decimalScale={8}
+      displayType='text'
+      renderText={(value) =>
+        <>
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/TokenAmountTextV2.tsx:30]: Do we need this container?

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/components/TokenAmountTextV2.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                style={tailwind('text-right')}
+                containerStyle={tailwind('justify-end')}
+                denominationCurrency={denominationCurrency}
+              />
+            )
+            : (
+              <ThemedTextV2
+                style={tailwind('text-xs font-normal-v2 text-right')}
+                dark={tailwind('text-mono-dark-v2-700')}
+                light={tailwind('text-mono-dark-v2-700')}
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/components/TokenAmountTextV2.tsx:51]:
```suggestion
                light={tailwind('text-mono-light-v2-700')}
```

File: mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx [hunk 3]
```diff
@@ -459,7 +459,6 @@ export function PortfolioScreen ({ navigation }: Props): JSX.Element {
     <View ref={containerRef} style={tailwind('flex-1')}>
       <ThemedScrollViewV2
         ref={ref}
-        light={tailwind('bg-gray-50')}
         contentContainerStyle={tailwind('pb-8')} testID='portfolio_list'
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Portfolio/PortfolioScreen.tsx:462]:
```suggestion
        contentContainerStyle={tailwind('pb-12')} testID='portfolio_list'
```

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**